src/crypto.cpp

```diff
@@ -297,102 +261,77 @@ int Crypto::AsymDec(
     return res;
 }
 
-
-int Crypto::SymEnc(
-    const uint8_t* sym_key,
-    const uint8_t* data,
-    const uint8_t* aad,
-    uint8_t*       enc_data,
-    size_t         data_size,
-    size_t         aad_size
-) {
+int Crypto::SymEnc(const uint8_t* sym_key, const uint8_t* data,
+                   const uint8_t* aad, uint8_t* enc_data, size_t data_size,
+                   size_t aad_size) {
     int res = -1;
 
     // Initialize GCM context
     mbedtls_gcm_context ctx;
     mbedtls_gcm_init(&ctx);
-    
+
     // Set the pointers so that the ciphertext is formatted as:
     //     IV || TAG || ENCRYPTED DATA
     uint8_t* iv = enc_data;
     uint8_t* tag = iv + CIPHER_IV_SIZE;
     uint8_t* output = tag + CIPHER_TAG_SIZE;
 
     // Add `sym_key` and AES cipher to the current GCM context
-    res = mbedtls_gcm_setkey(
-        &ctx,
-        MBEDTLS_CIPHER_ID_AES,
-        sym_key,
-        CIPHER_KEY_SIZE * 8); // Key size is given in bits
-    if( res != 0 ) {
+    res = mbedtls_gcm_setkey(&ctx, MBEDTLS_CIPHER_ID_AES, sym_key,
+                             CIPHER_KEY_SIZE * 8); // Key size is given in bits
+    if (res != 0) {
         mbedtls_gcm_free(&ctx);
-        spdlog::error("Failed to set symmetric key during symmetric key encryption.");
+        spdlog::error(
+            "Failed to set symmetric key during symmetric key encryption.");
         spdlog::error("Returned error: {}", to_string(res));
         return res;
     }
 
     // Sample randomness for the IV
     res = RandGen(iv, CIPHER_IV_SIZE);
-    if( res != 0 ) {
+    if (res != 0) {
         mbedtls_gcm_free(&ctx);
         spdlog::error("Failed to generate IV during symmetric key encryption.");
         spdlog::error("Returned error: {}", to_string(res));
         return res;
     }
 
     // Encrypt data
-    res = mbedtls_gcm_crypt_and_tag( 
-        &ctx,
-        MBEDTLS_GCM_ENCRYPT,
-        data_size,
-        iv,
-        CIPHER_IV_SIZE,
-        aad,
-        aad_size,
-        data,
-        output,
-        CIPHER_TAG_SIZE,
-        tag);
-    if( res != 0 ) {
+    res = mbedtls_gcm_crypt_and_tag(&ctx, MBEDTLS_GCM_ENCRYPT, data_size, iv,
+                                    CIPHER_IV_SIZE, aad, aad_size, data, output,
+                                    CIPHER_TAG_SIZE, tag);
+    if (res != 0) {
         mbedtls_gcm_free(&ctx);
         spdlog::error("Failed to perform symmetric key encryption.");
         spdlog::error("Returned error: {}", to_string(res));
     }
-    
+
     // Free the GCM context
     mbedtls_gcm_free(&ctx);
-    
+
     return res;
 }
 
-
-int Crypto::SymDec(
-    const uint8_t* sym_key,
-    const uint8_t* enc_data,
-    const uint8_t* aad,
-    uint8_t*       data,
-    size_t         enc_data_size,
-    size_t         aad_size
-) {
+int Crypto::SymDec(const uint8_t* sym_key, const uint8_t* enc_data,
```

Comment:
has this been formatted as well? i think we should add a linter to the CI to check styling. i moved S166 to the current sprint